Makefile

```diff
@@ -0,0 +1,12 @@
+build_release:
+	python3 ./release.py build
+
+release:
+	build_release
```

Comment:
Should this not be in the prerequisites of the `release` target?

```make
release: build_release
    python3 ./release.py publish ...
```

All in all I think this makefile is pretty dumb. There's no mechanism to tell whether a target's prerequisites have already been satisfied. That's the whole point of Make. And Make is not installed on windows by default. So it can just as well be a bash script.

Reply:
Agreed. Removed.